Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,21 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const maartjesTasks = tasks.map(elem => ({ ...elem, duration: elem.duration / 60 }));
+  const filteredTasks = maartjesTasks.filter(elem => elem.duration > 2);
+  const sumEarning = filteredTasks.reduce((acc, elem) => {
+    const earning = elem.duration * hourlyRate;
+    return acc + earning;
+  }, 0);
+  return sumEarning;
 }
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
 
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
+console.log(`Maartje has earned €${earnings.toFixed(2)}`);
```

Comment:
The output is `Maartje has earned €333.33`, but Maartje should actually be paid €373.33. Maartje is getting underpaid! Try to find the error in the code, hint; Maartje sometimes works for 2 hours.